presto-product-tests/bin/product-tests-suite-1.sh

```diff
@@ -6,7 +6,12 @@ suite_exit_code=0
 
 presto-product-tests-launcher/bin/run-launcher test run \
     --environment multinode \
-    -- -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds,cassandra,mysql,postgresql,kafka,hive_compression,"${DISTRO_SKIP_GROUP}" \
+    -- -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds,cassandra,mysql,postgresql,kafka,hive_compression,hive_cache"${DISTRO_SKIP_GROUP}" \
+    || suite_exit_code=1
+
+presto-product-tests-launcher/bin/run-launcher test run \
+    --environment multinode-hive-caching \
+    -- -g hive_cache,storage_formats \
```

Comment:
I think suite-1 is the longest. Can you confirm and pick a different suite?

Reply:
moved to suite 5